app/controllers/results_controller.rb

```diff
@@ -369,8 +397,15 @@ def codeviewer
       end
     end
 
-    @annots = @file.annotations
-    @all_annots = @file.submission.annotations
+    # if we are viewing an original result, we don't want
+    # to view annotations from any peer reviewers on those files
+    if !@result.is_a_review?
+      @annots = @file.annotations.select{|a| !User.find(a.creator_id).student?}
```

Comment:
This doesn't need to be this complex. Select annotations based on their associated result.